### PR TITLE
Adjust error message when trying to query JSON without mapping created

### PR DESCRIPTION
**Pull request by Holmistr**
Minor error message adjustment to make it clear whan you need to do when you want to query JSON objects.

Checklist:
- [x] Labels (`Team:`, `Type:`, `Source:`, `Module:`) and Milestone set
- [ ] Add `Add to Release Notes` label if changes should be mentioned in release notes or `Not Release Notes content` if changes are not relevant for release notes
- [ ] Request reviewers if possible
- [ ] New public APIs have `@Nonnull/@Nullable` annotations
- [ ] New public APIs have `@since` tags in Javadoc
- [ ] Send backports/forwardports if fix needs to be applied to past/future releases

### CODE REVIEW

**Review by Fly-Style — APPROVED** [review]
Looks awesome to me ;)

**Review by viliam-durina — APPROVED** [review]

**Review by gierlachg — APPROVED** [review]

**Comment by Holmistr** [comment]
Thank you for the reviews. Fixed the test, now waiting for the PR builder and if no objections, I will merge it after a green build.

**Comment by Holmistr** [comment]
run-lab-run

**Pull request merged**
